src/Sentry.Extensions.Logging/SentryLoggerProvider.cs

```diff
@@ -57,9 +57,16 @@ public SentryLoggerProvider(IOptions<SentryLoggingOptions> options, IHub hub)
                 _scope = hub.PushScope();
                 hub.ConfigureScope(s =>
                 {
-                    s.Sdk.Name = Constants.SdkName;
-                    s.Sdk.Version = NameAndVersion.Version;
-                    s.Sdk.AddPackage(ProtocolPackageName, NameAndVersion.Version);
+                    if (s.Sdk is {} sdk)
```

Comment:
Doesn't "CTRL + K + D" formats it like this?

```suggestion
                    if (s.Sdk is { } sdk)
```

I really would love for us to have an auto formatter that would apply as we save the file.
We'd run it on the whole solution first and then never have to pay attention at formatting again

Reply:
Yes, resharper doesn't seem to care if it's `{}` or `{ }`...